validator/client/wait_for_activation.go

```diff
@@ -150,15 +125,16 @@ func (v *validator) handleAccountsChanged(ctx context.Context, accountsChangedCh
 				valCount = int64(valCounts[0].Count)
 			}
 
-			valActivated := v.checkAndLogValidatorStatus(statuses, valCount)
-			if valActivated {
-				logActiveValidatorStatus(statuses)
-			} else {
+			if !v.checkAndLogValidatorStatus(statuses, valCount) {
 				continue
 			}
 		}
+		// If a validator is active, break out of this loop
 		break
 	}
+
+	// reset the ticker when they are all active
+	v.ticker = slots.NewSlotTicker(time.Unix(int64(v.genesisTime), 0), params.BeaconConfig().SecondsPerSlot)
```

Comment:
What's the point of this code? The ticker is already set here: https://github.com/prysmaticlabs/prysm/blob/2875ce6ee1dfa48ba10fdb1152ec6b2efa58464d/validator/client/validator.go#L282

Reply:
I removed it, i don't think it has a usecase if it's set elsewhere, i wish it was more clear where it was set however